install.json

```diff
@@ -1,54 +1,101 @@
 {
     "resources": {
-        "body": [
+        "head": [
```

Comment:
Including the files in the head allows the progress bar to appear earlier in the page's loading process.